test/parallel/test-stream-transform-objectmode-falsey-value.js

```diff
@@ -1,33 +1,30 @@
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 
 const stream = require('stream');
-var PassThrough = stream.PassThrough;
+const PassThrough = stream.PassThrough;
 
-var src = new PassThrough({ objectMode: true });
-var tx = new PassThrough({ objectMode: true });
-var dest = new PassThrough({ objectMode: true });
+const src = new PassThrough({ objectMode: true });
+const tx = new PassThrough({ objectMode: true });
+const dest = new PassThrough({ objectMode: true });
 
-var expect = [ -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-var results = [];
-process.on('exit', function() {
-  assert.deepStrictEqual(results, expect);
-  console.log('ok');
-});
+const expect = [ -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
+const results = [];
 
-dest.on('data', function(x) {
+dest.on('data', common.mustCall(function(x) {
   results.push(x);
-});
+}, expect.length));
 
 src.pipe(tx).pipe(dest);
 
-var i = -1;
-var int = setInterval(function() {
+let i = -1;
+const int = setInterval(function() {
   if (i > 10) {
```

Comment:
Would have to be something odd like `expect.length - 2` or `expect[expect.length -1]` or something like that. I'm not opposed but not exactly in favor either.